RecoTracker/CkfPattern/plugins/GroupedTrajCandLess.h

```diff
@@ -31,8 +31,9 @@ class dso_internal GroupedTrajCandLess
   template<typename T>
   float score (const T & t) const
   {
-
-    return t.chiSquared()-t.foundHits()*bonus+t.lostHits()*penalty 
+   auto bb = (t.dPhiCacheForLoopersReconstruction()==0 && t.foundHits()>8) ? 2*bonus : bonus; //extra bonus for long tracks not loopers
```

Comment:
and pt-cut  as well etc.
The point is : what's the use case?
Configurables are expensive, and their use makes sense only if they are supposed to be different for different workflow.

In any case as I said to say in the intro:
```
a future technical PR will try to optimize and factorize code (it will most probably recompile half tracking and tracking dependent code)
``` 
So this PR is mostly to be integrated in pre1 as is and its performance exposed to other POGs and PAGs. for feedback.